packages/editor-ui/src/n8n-theme.scss

```diff
@@ -1,7 +1,5 @@
 @import "./n8n-theme-variables";
 
-@import "~n8n-design-system/theme/dist/index.css";
```

Comment:
how are we importing this now?

Reply:
We're importing it directly in `design-system/src/main.js`. Doing so allows the build system to split into chunks and write font/asset paths properly.

https://github.com/n8n-io/n8n/pull/4061/files/0ab42166505e390ec27336f4c3e934eb5cac7c45#diff-2396dce48aeead825ea11d2267d6213773ebcb0887535809073d8ea228daed3b